styles.css

```diff
@@ -0,0 +1,163 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.app {
+    width: 100vw;
+    height: 100vh;
+    background-image: url("images/background.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.nav-bar {
+    max-width: 1440px;
+    height: 72px;
+    align-items: center;
+    display: flex;
+    padding: 0 20px;
+    justify-content:space-between;
+    margin: auto;
+}
+
+.nav-link-text {
+    font-size: 14px;
+    font-weight: 500;
+    line-height: 20px;
+    font-family: 'Inter', sans-serif;
+    color: #18191F;
+    text-decoration: none;
+}
+
+.nav-bar-list {
+    display: flex;
+    list-style: none;
+    gap: 32px;
+    align-items: center;
+}
+
+.nav-bar-list-right {
+    align-items: right;
+    display: flex;
+    list-style: none;
+    gap: 12px;
+}
+
+.hero-section {
+    display: flex;
+    margin: 100px auto auto;
+    max-width: 1440px;
+    padding: 0 20px; 
+    flex-direction: column;
+}
+
+.hero content {
+    gap: 32px;
+    display: flex;
+    flex-direction: column;
+}
+
+.hero-title {
+    font-family: 'Inter', sans-serif;
+    font-weight: 800px;
+    font-size: 72px;
+    line-height: 98px;
+    color: #18191f;
+    margin-top:70px;
+}
+
+.hero-button {
+    background-color:#8C30F5;
+    align-items: center;
+    padding: 12px 32px;
+    border-radius: 6px;
+    width: 154px;
+    height: 48px;
+    color: #FFFFFF;
+    border: 0;
+    font-family: 'Inter', sans-serif;
+    font-weight: 500px;
+    font-size: 16px;
+    line-height: 24px;
+    cursor: pointer;
+}
+
+.hero-content-wrapper {
+    display: flex;
+    justify-content: space-between;   
+}
+
+.hero-company-logos {
+    display: flex;
+    gap: 65px;
+    margin: auto;
+    align-items: center;
+    height: 110px;
+}
+
+.feature-section {
+    display: flex;
+    flex-direction: column;
+    height: 136px;
+    align-items: center;
+    gap: 0 8px;
+    
+}
+
+.feature-heading {
+    font-family: 'Inter', sans-serif;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 64px;
+    text-align: center;
+    color: #18191F;
+    margin: 80px auto auto;
+}
+
+.feature-p {
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 32px;
+    text-align: center;
+    color: #18191F;
+    padding: 0 300px;
+}
+
+.column-title {
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+    
+}
+
+.column {
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 26px;
+    text-align: center;
+}
+
+.content {
+    display: flex;
+    flex-direction: column;
+    gap: 120px;
+    justify-content: space-between;
+    margin-top: 120px;
+    align-items: center;
+}
+
+.row {
+    display: flex;
+    gap: 30px;
+}
+
+.icons {
+    text-align: center;
+    margin: 0 250px;
```

Comment:
- [ ] This margin is not necessary here. Use flexbox to align the items to the center.